### ISSUE TITLE
Installing Robyn in databricks - rlang package version issue

### ISSUE THREAD

**Issue by Deepti0326** [facebookexperimental/Robyn#1189]
## Describe issue
I am trying and installing Robyn in databricks, It was running fine for few days, when I tried and installed it again, I am getting an issue telling the version of rlang is lower. I tried installing that package separately and installed Robyn afterwards also, but then also the same issue persists.

![image](https://github.com/user-attachments/assets/f46dc5d3-22df-40af-8ee0-6059bf7a8dbb)

**Comment by gufengzhou** [comment]
have you checked this? https://stackoverflow.com/questions/65960432/rlang-not-updating-to-latest-in-databricks

**Comment by Deepti0326** [comment]
> have you checked this? https://stackoverflow.com/questions/65960432/rlang-not-updating-to-latest-in-databricks

Yes, I did. I installed the "rlang" package first and then installed "Robyn". Now the error is not coming anymore. Thank you
